app/components/doi-size.js

```diff
@@ -0,0 +1,27 @@
+import Component from '@ember/component';
+// import { set } from '@ember/object';
+
+export default Component.extend({
+
+  updateSize(size) {
+    this.set('fragment', size);
```

Comment:
I am having a problem updating Array attributes that are not Fragments. For example, each fragment/element in the Sizes property is just a string. Updating the property of the component works, but the attribute (`fragment` in this case) that contains that should be binding to that property does not. 

as a consequence i can see values being inputed and rendered correctly but i can't add or modify values.